varats-core/varats/report/gnu_time_report.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, path: Path) -> None:
                         TimeReport._parse_wall_clock_time(line)
                     continue
 
-                print("Not matched: ", line)
+                # print("Not matched: ", line)
```

Comment:
```suggestion
                print("Not matched: ", line)
```
Left over comment or should we remove this line?

Reply:
I'd say let's remove it. For example, when parsing 100 time reports, stdout will be full of 'Not matched'.